sass_processor/management/commands/compilescss.py

```diff
@@ -83,6 +83,13 @@ def add_arguments(self, parser):
             help=_(
                 "Set the precision for numeric computations in the SASS processor. Default: settings.SASS_PRECISION.")
         )
+        parser.add_argument(
+            '--exclude-py',
+            action='store_true',
+            dest='exclude_py',
+            default=False,
```

Comment:
This default is unnecessary because of the `store_true` action.

Reply:
The other parser args with "store_true" have that (--delete-files and --use-processor-root). Should I remove it there as well @AndreasBackx? Thanks for the comment!

Reply:
Sorry didn't see that. I would leave it up to @jrief to decide. Should be fine for me.

Reply:
This should be consistent for all args, and if we decide that all args have unnecessary options, this should be refactored in a separate PR.